codelite_vim/vimCommands.cpp

```diff
@@ -8,7 +8,7 @@
 /*EXPERIMENTAL*/
 #include "wx/uiaction.h"
 
-VimCommand::VimCommand()
+VimCommand::VimCommand(IManager* m_mgr)
```

Comment:
Just an FYI:
You can call `clGetManager()` (defined in `globals.h`) to get the instance of the `IManager` without passing it around